splunkforwarder/elx/init.sls

```diff
@@ -11,6 +11,20 @@
 {%- from tpldir ~ '/map.jinja' import splunkforwarder with context %}
 
 {%- for port in splunkforwarder.client_out_ports %}
+  {%- if salt.grains.get('osmajorrelease') == '7'%}
+    {%- set FwZone = salt.firewalld.default_zone() %}
+Allow Splunk Mgmt Outbound Port {{ port }}:
+  module.run:
+    - name: 'firewalld.add_port'
```

Comment:
Ok, cool. Just wanted to doublecheck cuz that's not always the case when relying on modules directly. (Often the state implements the stateful logic, and the module just attempts to do what you tell it to do.)